{"uuid":"a026e6d6-3a9f-4a47-b60f-379b4a05878b","name":"Scenario Outline: The user getting error message with invalid python code for Traversal page","children":["d44a80e4-0707-4ed9-b97f-304e370eb6bc"],"befores":[],"afters":[{"name":"Hooks.Hooks.a(io.cucumber.java.Scenario)","status":"passed","stage":"finished","steps":[],"attachments":[{"name":"My screenshot","source":"d2f7c69e-0000-4169-aa69-0d7ed4d897f1-attachment","type":"image/png"},{"name":"Myscreenshot","source":"5a55c363-5801-4290-bea0-faa00eb86331-attachment"}],"parameters":[],"start":1696885867087,"stop":1696885867362},{"name":"Hooks.Hooks.a(io.cucumber.java.Scenario)","status":"skipped","statusDetails":{"known":false,"muted":false,"flaky":false,"message":"AFTER_STEP is failed."},"stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1696885867362,"stop":1696885867362},{"name":"Hooks.Hooks.a(io.cucumber.java.Scenario)","status":"skipped","statusDetails":{"known":false,"muted":false,"flaky":false,"message":"AFTER_STEP is failed."},"stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1696885867364,"stop":1696885867364},{"name":"Hooks.Hooks.a(io.cucumber.java.Scenario)","status":"skipped","statusDetails":{"known":false,"muted":false,"flaky":false,"message":"AFTER_STEP is failed."},"stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1696885867365,"stop":1696885867365}],"start":1696885867084,"stop":1696885867367}